db/migrate/20200320104040_preupgrade_report_entries.rb

```diff
@@ -2,6 +2,7 @@ class PreupgradeReportEntries < ActiveRecord::Migration[5.2]
   def up
     create_table :preupgrade_report_entries do |t|
       t.integer :preupgrade_report_id, null: false
+      t.string :host_id, null: false
```

Comment:
since the plugin hasn't been released it's acceptable, however what's the reason to modify existing migration and not adding a new one? note that other developers already migrated this in their DBs, they will need to manually fix their DB if this is merged.

Reply:
Good point, I'll add it as migration.